### PR TITLE
chore(I3S-Debug): memory stats positioning change

### PR DESCRIPTION
**Pull request by morphoix**
Screenshot from 2021-07-30 16-57-27

### CODE REVIEW

**Review by ibgreen — APPROVED** [review]
UI is starting to look very polished. 

Note that there is a bug in the memory stats formatting. Probably in probe.gl. Would be great to fix that.

I think the labels could also be improved. Try showing them to someone that doesn't know what the app does...

Maybe something like:
- "Show" -> "Minimap"
- "Use Different Viewports" -> ??? What does this do?
- "Pickable" -> What is the point of letting user disable picking? Switching between object and tile picking could have made more sense? I forgot why this switch is here.
- "UV DEBUG TEXTURE" -> "TEXTURE UVs"
- "WIREFRAME MODE" -> "WIREFRAME"
- "BOUNDING VOLUMES" -> None/Sphere/Box

**Comment by belom88** [comment]
>"Use Different Viewports" -> ??? What does this do?

* The mini-map loads tiles from its viewport state with this option, instead of showing tiles of the main viewport
* For showing main viewport tiles `Scatterplot` is shown on the mini-map

>"Pickable" -> What is the point of letting user disable picking?

* Tiles are highlighted onHover. It's looks to noisy when picking is continually switched on.
* Switching between object and tile picking make sense. We didn't have time to do it during the contract. It requires extra efforts.

**Comment by morphoix** [comment]
> * "Show" -> "Minimap"
> * "Use Different Viewports" -> ??? What does this do?
> * "Pickable" -> What is the point of letting user disable picking? Switching between object and tile picking could have made more sense? I forgot why this switch is here.
> * "UV DEBUG TEXTURE" -> "TEXTURE UVs"
> * "WIREFRAME MODE" -> "WIREFRAME"
> * "BOUNDING VOLUMES" -> None/Sphere/Box

@ibgreen As bounding volumes have also select by color my suggestion is to leave the toggle but remove both of selects from a screen when it's not used

Screenshot from 2021-07-30 17-24-26
Screenshot from 2021-07-30 17-09-11
Screenshot from 2021-07-30 17-04-04

File: examples/website/i3s/components/control-panel.js [hunk 1]
```diff
@@ -5,104 +5,115 @@ import {EXAMPLES} from '../examples';
 import {MAP_STYLES} from '../constants';
 
 const Container = styled.div`
-  display: flex;
-  flex-direction: column;
+  display: grid;
+  grid-template-columns: 100px 100px 100px;
```
Comment (belom88) [inline examples/website/i3s/components/control-panel.js:9]: grid isn't supported by IE

**Review by belom88 — CHANGES_REQUESTED** [review]
@ibgreen  do we need the website supported on IE11?

**Comment by dryabinin-actionengine** [comment]
@morphoix Also debug texture picture should be larger in debug panel. User should be able to compare picture with debug texture on the tileset.

**Comment by morphoix** [comment]
> @morphoix Also debug texture picture should be larger in debug panel. User should be able to compare picture with debug texture on the tileset.

hover effect added

File: examples/website/i3s/app.js [hunk 2]
```diff
@@ -214,14 +218,11 @@ export default class App extends PureComponent {
...
-    const {viewState, selectedMapStyle, selectedFeatureAttributes, showMemory} = this.state;
+    const {viewState, selectedMapStyle, selectedFeatureAttributes} = this.state;
 
     return (
       <div style={{position: 'relative', height: '100%'}}>
-        <MemoryButton onClick={() => this.setState({showMemory: !showMemory})}>{showMemory ? `Hide` : `Show`} Memory Usage</MemoryButton>
-        <StatsWidgetWrapper showMemory={showMemory}>
-          {this._renderStats()}
-        </StatsWidgetWrapper>
+
```
Comment (belom88) [inline examples/website/i3s/app.js:225]: Nit: remove empty line

File: examples/website/i3s/components/debug-panel.js [hunk 3]
```diff
@@ -86,27 +94,28 @@ const CheckboxOption = styled.div`
...
 
 const propTypes = {
-  children: PropTypes.object,
   isClearButtonDisabled: PropTypes.bool,
   onDebugOptionsChange: PropTypes.func,
   clearWarnings: PropTypes.func,
   debugTextureImage: PropTypes.string,
-  debugOptions: PropTypes.object
+  debugOptions: PropTypes.object,
+
```
Comment (belom88) [inline examples/website/i3s/components/debug-panel.js:118]: Nit: remove empty line

**Review by belom88 — APPROVED** [review]

**Pull request merged**